controllers/comments.controllers.js

```diff
@@ -70,4 +71,20 @@ exports.getCommentsByArticleId = ({ params: { article_id } }, res, next) => {
         }
       })
       .catch(next);
-  };
\ No newline at end of file
+  };
+
+exports.srvDeleteCommentByCommentId = ({params : {comment_id}},res,next) => {
+
+  return dbDeleteCommentByCommentId(comment_id)
+  .then(({rows : deletedCommentRow})=>{
+    if(deletedCommentRow.length){
+      res.status(204).send();
+    }else{
+      return Promise.reject({
```

Comment:
I would move this condition and custom error to the model.  This is where we should check/format/prepare our data that we're sending back.

Reply:
Ah, thanks, I was feeling like the controller code was getting busy where it didn't want to be so busy... I'll refactor that after I've done with the task13 hosting setup.